sympy/logic/boolalg.py

```diff
@@ -360,13 +360,13 @@ def as_set(self):
 
 class BooleanFalse(BooleanAtom, metaclass=Singleton):
     """
-    SymPy version of False, a singleton that can be accessed via S.false.
+    SymPy version of ``False``, a singleton that can be accessed via ``S.false``.
 
-    This is the SymPy version of False, for use in the logic module. The
-    primary advantage of using false instead of False is that shorthand boolean
-    operations like ~ and >> will work as expected on this class, whereas with
-    False they act bitwise on 0. Functions in the logic module will return this
-    class when they evaluate to false.
+    This is the SymPy version of ``False``, for use in the logic module. The
+    primary advantage of using ``false`` instead of ``False`` is that shorthand
+    Boolean operations like ``~`` and ``>>`` will work as expected on this class,
+    whereas with ``False`` they act bitwise on 0. Functions in the logic module
+    will return this class when they evaluate to false.
```

Comment:
It also supports all the other Basic methods like `subs` etc. That's the real reason it is needed: Basic args need to be Basic.

Reply:
I was not really considering exactly what it said, focused on the formatting... ;-)